### PR TITLE
update results section

### PR DESCRIPTION
**Pull request by dmurvihill**

### CODE REVIEW

File: doc/src/results.ltx [hunk 1]
```diff
@@ -184,31 +184,32 @@ During development, \textit{Checksims} was used in real-world situations on two
...
 teaching assistant. This assignment was an excercise in optimization and
-bitwise operations, and was composed of a relatively small C project with a
-great deal of common code. \textit{Checksims} detected three students with
-extremely similar submissions who would not otherwise have been identified,
-given that the assignment was graded programmatically, with very little human
-interaction between course staff and student submissions.
+bitwise operations and was composed of a relatively small C project with a
+great deal of common code. In a class of 68, \textit{Checksims} detected three
+students with extremely similar submissions. Without \textit{Checksims}, It is
+unlikely that the similarity would have been detected because the grading
```
Comment (mheon) [inline doc/src/results.ltx]: The because feels awkward. "Given that" instead? I think a comma goes there too, it's two independent clauses.

**Comment by dmurvihill** [comment]
"Given that" feels way more awkward to me. I will have another go at it. Independent clauses do not always need to be separated by a comma, especially if one is subordinate.

File: doc/src/results.ltx (hunk 1 above)
Comment (dmurvihill) [inline doc/src/results.ltx]: See if this is better.

**Comment by mheon** [comment]
Alright, looks better

**Pull request merged**